test/fixtures/pvc/storage_class.yml

```diff
@@ -0,0 +1,6 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: k8s-deploy-test
+provisioner: kubernetes.io/no-provisioner
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
Since this field is important to the logic of the program, it makes sense to reflect its value in the filename.
Alternatively, leave this field blank and add it dynamically when you load the fixture.